### PR TITLE
adds defaults, registration  and initialization

### PR DESCRIPTION
**Pull request by anthonymartin**

### CODE REVIEW

File: packages/contracts/contracts/GrantElections.sol [hunk 1]
```diff
@@ -15,39 +16,113 @@ contract GrantElections {
...
+        beneficiaryRegistry = _beneficiaryRegistry;
+
+        _setDefaults();
+    }
+
+    // todo: mint POP for caller to incentivize calling function
+    function initialize(ElectionTerm _grantTerm) public {
+        uint8 _term = uint8(_grantTerm);
+        Election storage _election = elections[_term];
+        require(_election.exists && _election.electionState == ElectionState.Closed, "election can't be started yet");
```
Comment (miloops) [inline packages/contracts/contracts/GrantElections.sol:66]: does it have to be closed to be started?

**Review by miloops — APPROVED** [review]
looks good to me, just a comment/question

File: packages/contracts/contracts/GrantElections.sol (hunk 1 above)
Comment (anthonymartin) [reply packages/contracts/contracts/GrantElections.sol:66]: @miloops yes it does. you don't want to overwrite an existing election if the previous one has not finished

**Comment by Amirjab21** [comment]
This looks good to me - is it ready @anthonymartin ?

**Pull request merged**